playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -295,6 +295,7 @@
   vars:
     _component_runcommand: "./tools/deployment/multinode/100-glance.sh"
     _component_run_env:
+      OSH_OPENSTACK_RELEASE: "rocky"
```

Comment:
should we use a generic var on commo-vars.yml to track this in case we need it somewhere else? I dont like to hardcode things.

Reply:
Good point